src/main/scala/outwatch/dom/DomTypes.scala

```diff
@@ -14,11 +14,11 @@ import org.scalajs.dom
 import helpers._
 
 private[outwatch] object DomTypesBuilder {
-  type GenericVNode[T] = VNode_
+  type GenericVNode[T] = VTree
 
-  trait VNodeBuilder extends TagBuilder[GenericVNode, VNode_] {
+  trait VNodeBuilder extends TagBuilder[GenericVNode, VTree] {
     // we can ignore information about void tags here, because snabbdom handles this automatically for us based on the tagname.
-    override def tag[Ref <: VNode_](tagName: String, void: Boolean): VNode_ = VTree(tagName, Seq.empty)
+    protected override def tag[Ref <: VTree](tagName: String, void: Boolean): VTree = VTree(tagName, Seq.empty)
```

Comment:
When we make this protected, we need to introduce a `def tag` in OutwatchAttributes for as we did for `attr`, `prop`, `style`. Then we should also add a testcase for custom tags.